.travis.yml

```diff
@@ -78,6 +79,11 @@ jobs:
             php: 5.6
             env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
 
+        -   <<: *STANDARD_TEST_JOB
+            stage: Test
+            php: 7.1
+            env: SYMFONY_DEPRECATIONS_HELPER=disabled PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1 SYMFONY_VERSION="~3.0"
```

Comment:
why `SYMFONY_DEPRECATIONS_HELPER=disabled` ?
why `PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1` ?

Reply:
Job not used anymore, using `COMPOSER_FLAGS="--prefer-stable --prefer-lowest"` instead which already installs Symfony 3 components